registry.json

```diff
@@ -5403,5 +5403,20 @@
     "telegram": "caikejungle",
     "operatorName": "Carlos Souza",
     "github": "wowica"
+  },
+   {
+    "ticker": "PUFF",
+    "poolId": "ff5940e0061e5c8c3e37f76c47bd7be5ab8eb9dfe43f6c2d1b74f028",
+    "social": {
+      "website": "https://puffinpool.com",
+      "twitter": "",
```

Comment:
There is no social medial presence for the pool. Please add some

Reply:
Does discord not count as a social?

Reply:
I'll add a twitter account shortly